cmd/spec/path.yaml

```
@@ -153,4 +153,31 @@ paths:
                 $ref: '#/components/schemas/v1.InternalServerError'
           description: There was an internal server error.
       summary: Get Image OSTree repo.
-      description: Returns the information of the OSTree Repository of a particular Image.
\ No newline at end of file
+      description: Returns the information of the OSTree Repository of a particular Image.
+  /updates:
+    post:
+      operationId: Update a device
```

```suggestion
      operationId: updateDevice
```

Just to keep the operationId's simple, they're used on the URLs